next-sitemap.config.js

```diff
@@ -4,9 +4,47 @@
  */
 
 /** @type {import('next-sitemap').IConfig} */
+const siteUrl = process.env.SITE_URL || 'https://developer.hashicorp.com'
+
 module.exports = {
-	siteUrl: process.env.SITE_URL || 'https://developer.hashicorp.com',
-	generateRobotsTxt: true, // (optional)
-	exclude: ['/swingset*', '/onboarding/*', '/profile*'],
-	// ...other options
+	siteUrl,
+	generateRobotsTxt: true,
+	robotsTxtOptions: {
+		additionalSitemaps: [
+			`${siteUrl}/hcp-sitemap.xml`,
+			`${siteUrl}/server-sitemap.xml`,
+		],
+	},
+	exclude: [
```

Comment:
Maybe we can add a comment here explaining why we have the extensive exclude list 💭

Reply:
Great idea, will add in